lib/src/layer/polygon_layer/polygon.dart

```diff
@@ -31,9 +31,11 @@ class Polygon<R extends Object> {
   /// Defaults to false (enabled).
   final bool disableHolesBorder;
 
-  /// Set to true if the border of the [Polygon] should be rendered
-  /// as dotted line.
-  final bool isDotted;
```

Comment:
We need to deprecate this, along with the equivalent in `Polyline`.
I originally did this for `Polyline`s in 72b1a51 after we discussed it, but it was removed in 9c62133 - is there any reason the method I added can't be used?

Reply:
@JaffaKetchup No problem, it's just that in other projects I use the opportunity of (potentially unrelated) "breaking changes" to remove deprecated code almost everywhere: cleaner code.
If you prefer to keep for a moment those deprecated fields, please put them back.

Reply:
Yeah, I suppose that makes more sense. We've discussed internally, and we'll move towards that.